styleguide/css/main.scss

```diff
@@ -1,31 +1,72 @@
-body {
-  margin-top: 60px;
+$white: #fff;
+$light-gray-white: #efefef;
+$dark-gray: #333;
+$medium-gray: #666;
+
+
+.mainContainer {
+  padding: 40px 0;
+  transition: margin-left .5s;
```

Comment:
I personally would prefer using `transform: translateX();`. 

It's much smoother than transitions on position properties. 

See: https://css-tricks.com/almanac/properties/t/transform/